balancer/weightedroundrobin/balancer.go

```diff
@@ -236,7 +233,7 @@ func (b *wrrBalancer) UpdateClientConnState(ccs balancer.ClientConnState) error
 	// This causes child to update picker inline and will thus cause inline
 	// picker update.
 	return b.child.UpdateClientConnState(balancer.ClientConnState{
-		BalancerConfig: endpointShardingLBConfig,
+		BalancerConfig: nil, // pickfirst can handle nil configs.
```

Comment:
And then I'd delete it from all our code to avoid clutter.  Most LB policies should handle not having configs, except some special xds things.